assets/css/admin/addon-list.css

```diff
@@ -0,0 +1,58 @@
+.el-container {
+	width: 100%;
+}
+
+.clear {
```

Comment:
It is there in `wp-admin/css/common.css` file. Here is the link to it https://github.com/WordPress/WordPress/blob/master/wp-admin/css/common.css#L109